modules/uac_auth/auth.c

```diff
@@ -285,7 +285,7 @@ void uac_calc_HA1( struct uac_credential *crd,
 	MD5Update(&Md5Ctx, crd->passwd.s, crd->passwd.len);
 	MD5Final(HA1, &Md5Ctx);
 
-	if ( auth->flags& AUTHENTICATE_MD5SESS )
+	if ( auth->algorithm == ALG_MD5SESS )
```

Comment:
Please squash this fix-commit into the main commit of this PR.  You can use `git rebase -i` and merge them, then force-push to your branch in order to obtain this effect.  Alternatively, you can `git reset HEAD~`, then `git add modules && git commit --amend`, then force-push to your branch.

Reply:
Tried git rebase -i as suggest, not sure if it helped a lot, I see 4 commits now. :(